.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,83 @@
+name: Bug Report
+description: Report an issue with Everest.
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Thanks for reporting a bug!
+  - type: markdown
+    attributes:
+      value: '## Please confirm:'
+  - type: input
+    attributes:
+      label: My mods and Everest install are up to date
+      placeholder: yes/no
+    validations:
+      required: true
```

```suggestion
  - type: dropdown
    attributes:
      label: My mods and Everest install are up to date
      options:
        - 'Yes'
        - 'No'
    validations:
      required: true
```

These can be changed to dropdowns if you want a Y/N option but don't prefer using checkboxes

---

Good idea, thanks!